app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryUiRenderer.kt

```diff
@@ -28,7 +28,10 @@ class BloodSugarEntryUiRenderer(
       Random, PostPrandial, Fasting -> {
         setInputTypeUnitPreferenceSelectionBased(model)
       }
-      HbA1c -> ui.decimalOrNumericBloodSugarInputType()
+      HbA1c -> {
+        ui.decimalOrNumericBloodSugarInputType()
+        ui.setLabelForHbA1c()
+      }
       else -> ui.numericBloodSugarInputType()
```

Comment:
If we are going to hide the unit preference button in unknow measurement type, we should be showing the label as `mg/dL`. WDYT?

Reply:
Yes I think mg/dL is better than showing nothing there.